ui/shared/components/form/XHRUploaderWithEvents.jsx

```diff
@@ -46,6 +48,10 @@ class XHRUploaderWithEvents extends XHRUploader {
     )
   }
 
+  renderButton() {
```

Comment:
currently, the default logic in the base component renders an empty space for an "upload" button that we do not show, this instead uses that space to show an error message if relevant

Reply:
this class extends an imported library, so it is overriding the base class implementation